src/commands/emasser/hello.ts

```diff
@@ -6,7 +6,8 @@ export default class EmasserSayHello extends Command {
   static hidden = true
 
   async run(): Promise<void> { // skipcq: JS-0116, JS-0105
-    let user = 'rookie'
+    const users = ['rookie', 'greenhorn', 'novice', 'expert', 'oracle', 'maestro']
```

Comment:
😆

Reply:
wonder how many folks will discover this Easter egg...lol